icubam/backoffice/handlers/base.py

```diff
@@ -81,14 +81,16 @@ def parse_from_body(self, cls) -> dict:
     """Given a store class, parse the body a dictionary."""
     result = dict()
     for col in cls.get_column_names():
-      value = self.get_body_arguments(col + '[]', None)
+      value: Union[
+        Optional[str],
+        List[str]] = self.get_body_arguments(col + '[]', strip=False)
```

Comment:
This one is strange. It have been inferred from `get_body_arguments` output. Is it well annotated?

Reply:
To be honest, I think this code would deserve some explanations, not just type annotations ;)

Reply:
I don't know how type inference in `mypy` works, but at least this cannot be guessed locally, since we then assign `get_body_argument` (note the singular). They return different types.

It's not trivial for me to figure out the intention behind this code, so maybe @olivierteboul can give some explanations, then I can add a comment and improve the annotations if possible.

Reply:
OK, I guess we can leave it as is for now and improve later.